charts/sequencer/files/cometbft/config/custom.genesis.json

```diff
@@ -168,7 +168,7 @@
       },
       "oracle": {
         "currencyPairGenesis": [
-          {{- range $index, $genesis := .Values.sequencer.priceFeed.currencyPairGenesis }}
+          {{- range $index, $genesis := .Values.genesis.priceFeed.currencyPairGenesis }}
```

Comment:
it was unclear why these values were defined on the sequencer when only used in genesis or upgrade